docs/getting-started/self-healing-rollback.asciidoc

```diff
@@ -0,0 +1,20 @@
+[[self-healing-rollback]]
+= Configure self-healing rollback for Windows endpoints
+
+{endpoint-cloud-sec}'s self-healing feature rolls back file changes and processes on Windows endpoints when a prevention alert is generated by enabled protection features. All activity on the host reverts to its state five minutes before the prevention alert.
```

Comment:
Process remediation is planned but not implemented yet. Also, afaik there is nuance with the 5 min value. For example, modified files are rollbacked to most recent snapshot, which can be much longer than 5 minutes

Reply:
Thanks @joe-desimone for the extra info! Is this a standard Elasticsearch snapshot, or something unique to Endpoint? And can we say anything more about how often the snapshots are taken, what triggers them, etc.? To give readers a sense of what to expect.

Reply:
These snapshots are unique to endpoint. We use built in volume shadow copy functionality to windows https://docs.microsoft.com/en-us/windows-server/storage/file-server/volume-shadow-copy-service. I believe we take a snapshot every 2 hours. We might tweak this slightly or even make it user configurable in the future though

Reply:
Cc @bit-envoy

Reply:
@joe-desimone @bit-envoy Wondering if this covers it -- I don't think we need too much detail about Endpoint's inner workings, but a general time window might help users know how far back to expect rollback, for example to confirm any incidental changes:
```suggestion
{endpoint-cloud-sec}'s self-healing feature rolls back file changes on Windows endpoints when a prevention alert is generated by enabled protection features. File changes on the host revert to their state up to two hours before the prevention alert.
```

Reply:
I think this is accurate but it would be best if @bit-envoy can confirm

Reply:
It is reverting all changes that happened in five minutes to disk state that was captured during taking snapshot (which might max 2h)

Reply:
Thanks, @bit-envoy! Is it accurate to say that it rolls back five minutes _from when the alert was created_? Or is there another reference point for the five minutes?